code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -84,3 +84,5 @@
 	var/hal_screwydoll
 	/// When an braindead player has their equipment fiddled with, we log that info here for when they come back so they know who took their ID while they were DC'd for 30 seconds
 	var/list/afk_thefts
+	/// A list to keep track of which books a person has read (to prevent people from reading the same book again and again for positive mood events)
+	var/list/book_titles_read
```

Comment:
Seems more fitting as a list on `/mind`, instead of `/human`? IMO.

Reply:
Makes sense.  Applied with https://github.com/tgstation/tgstation/pull/66371/commits/6b368085131542256fdf33d0f62e27cb6abc82d1